programs/native/bpf_loader/src/lib.rs

```diff
@@ -88,13 +88,18 @@ pub fn helper_sol_log_u64(arg1: u64, arg2: u64, arg3: u64, arg4: u64, arg5: u64)
     0
 }
 
-fn create_vm(prog: &[u8]) -> Result<EbpfVmRaw, Error> {
+pub fn helper_sol_tick_height(_arg1: u64, _arg2: u64, _arg3: u64, _arg4: u64, _arg5: u64) -> u64 {
+    0 // TODO!  Get tick_height somehow...
```

Comment:
Right now Ebpf helper functions can have no context as they are just function pointers.  I think I'd like to modify Ebpf to take a Fn trait instead so we can have closures.  @jackcmay wdyt?

Reply:
I went with option 2, extending `sol_deserialize()`

Reply:
I'll be digging into the jit implementation as part of loop caps and as part of monetizing instructions.  Will look at adding closures then.  For now, passing in tick_hight seems ok.